com.stansassets.build/Editor/Core/BuildExecutor.cs

```diff
@@ -44,7 +42,7 @@ public static void Build(BuildContext buildContext)
         {
             s_BuildContext = buildContext;
 
-            OnBuidExecutorRegistrationStarted.Invoke(s_BuildContext);
+            RegisterAllIBuildExecutorListener(buildContext);
```

Comment:
We already calling this method inside BuildExecutor class so I don't see any reason to point that inside method name. (Register~~AllIBuildExecutor~~Listener)